ios/FluentUI/People Picker/ContactCollectionView.swift

```diff
@@ -0,0 +1,138 @@
+        static let leadingInset: CGFloat = 16.0
+    }
+
+    private let layout: ContactCollectionViewLayout
+    private var widthConstraint: NSLayoutConstraint?
+    private lazy var heightConstraint: NSLayoutConstraint = {
+        let heightConstraint = heightAnchor.constraint(equalToConstant: 0.0)
+        heightConstraint.isActive = true
+        return heightConstraint
+    }()
+}
+
+extension ContactCollectionView: UICollectionViewDataSource {
+    @objc public func collectionView(_ collectionView: UICollectionView, numberOfItemsInSection section: Int) -> Int {
+        return contactList.count
+    }
+
+    @objc public func collectionView(_ collectionView: UICollectionView, cellForItemAt indexPath: IndexPath) -> UICollectionViewCell {
+        let cell = collectionView.dequeueReusableCell(withReuseIdentifier: ContactCollectionViewCell.identifier, for: indexPath) as! ContactCollectionViewCell
+        cell.setup(contact: contactList[indexPath.item])
+
+        return cell
+    }
+}
+
+extension ContactCollectionView: UICollectionViewDelegate {
+    // If the Contact is not fully visible in the scroll frame, scrolls the view by an offset large enough
+    // so that the tapped Contact is fully visible.
+    @objc public func collectionView(_ collectionView: UICollectionView, didSelectItemAt indexPath: IndexPath) {
+        guard let cell = cellForItem(at: indexPath) else {
+            return
+        }
+
+        let cellWidth = cell.frame.width
+        let cellFrame = cell.contentView.convert(cell.bounds, to: self)
+        let cellLeftPosition = cellFrame.origin.x
+        let cellRightPosition = cellLeftPosition + cellWidth
+        let viewLeadingPosition = bounds.origin.x
+        let viewTrailingPosition = viewLeadingPosition + frame.size.width
+
+        let extraScrollWidth = layout.minimumLineSpacingForSectionAt(section: 0)
+        var offSet = contentOffset.x
+        if cellLeftPosition < viewLeadingPosition {
+            offSet = cellLeftPosition - extraScrollWidth
+            offSet = max(offSet, -Constants.leadingInset)
+        } else if cellRightPosition > viewTrailingPosition {
+            let maxOffsetX = contentSize.width - frame.size.width + extraScrollWidth
+            offSet = cellRightPosition - frame.size.width + extraScrollWidth
+            offSet = min(offSet, maxOffsetX)
+        }
+
+        if offSet != contentOffset.x {
+            setContentOffset(CGPoint(x: offSet, y: contentOffset.y), animated: true)
+        }
+    }
+}
+
+extension UIContentSizeCategory {
+    var contactHeight: CGFloat {
+        switch self {
+        case .extraSmall:
+            return 115.0
+        case .small:
+            return 117.0
+        case .medium:
+            return 118.0
+        case .large:
+            return 121.0
+        case .extraLarge:
+            return 125.0
+        case .extraExtraLarge:
+            return 129.0
+        case .extraExtraExtraLarge:
+            return 135.0
+        default:
```

Comment:
let's not have default defined so it will show compiler error when new enum is created

Reply:
Got it!

Reply:
yes

Reply:
Just to confirm, we're keeping the default right?

Reply:
no remove default. we should be explicit for all the other cases thatit will use .extraExtraExtraLarge by design.

Reply:
After discussing with @MathieuKavalec, there seems to be an issue with Xcode not correctly understanding that a switch statement is already exhaustive and still requires a `default` (otherwise building will fail). Due to this, I've grouped the `.unspecified` category into `default`.

Currently there aren't any redlines for the accessibility sizing, so I've grouped them under the `.extraExtraExtraLarge` case since those have the closest height. Perhaps we can update the switch statement when designs for the missing sizes are finalized?